client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -404,7 +405,7 @@ class EmailProvidersComparison extends Component {
 		const discount = productIsDiscounted ? (
 			<span className="email-providers-comparison__discount-with-renewal">
 				{ translate(
-					'%(discount)d% Off{{span}}, %(discountedPrice)s billed today, renews at %(standardPrice)s{{/span}}',
+					'%(discount)d% off{{span}}, %(discountedPrice)s billed today, renews at %(standardPrice)s{{/span}}',
```

Comment:
For Titan we show `3 months free` (without any uppercase) so I felt we should be consistent here as well.